pages/common/piodebuggdb.md

```diff
@@ -0,0 +1,7 @@
+# piodebuggdb
+
+> This command is an alias of `pio debug --interface={{gdb}}`.
```

Comment:
👌🏻

Reply:
I just saw that the node client doesn't render `{{ }}` correctly, if it's not used where it expects it to be. 😢

Reply:
Actually, let's remove the tokens.  We're just stating _what_ `piodebuggdb` is an alias for—and the user can't change that, hence there's no need to the token syntax.
```suggestion
> This command is an alias of `pio debug --interface=gdb`.
```

Reply:
You're right. I just like the colorful output `{{ }}` creates.

Reply:
Yeah, I do to :)